extensions/amp-date-picker/0.1/defaultPhrases.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const calendarLabel = 'Calendar';
```

Comment:
hmm, do we need to provide this to the react dates? Don't they have this builtin that works with `locale`? worried about l11n...

Reply:
The builtin locale support is only for the date values, not the a11y messages. I plan to add support for publisher-defined phrases, but I think it will be a nicely isolated task to complete after this MVP is merged.  I'll open an issue for it. What do you think?

Reply:
#13747